packages/fluentui/react-northstar/test/specs/components/Datepicker/Datepicker-test.tsx

```diff
@@ -0,0 +1,5 @@
+import Unstable_Datepicker from 'src/components/Datepicker/Datepicker';
+
+describe('Datepicker', () => {
+  it('filler_test', () => expect(Unstable_Datepicker.displayName).not.toBeNull());
```

Comment:
`isConformant`?

Reply:
Currently, we don't run the isConformant test as the component is exported as Unstable_.
The isConformant naming convention is not prepared for this (className, constructorName, docBlock descriptions).

The decision was then to ignore the isConformant test and only run it locally during development.

FYI: @pompomon

Reply:
Yep, we've agreed to skip isConformant while the component is in "Unstable_*" state